problems/2.functions.js

```diff
@@ -2,15 +2,18 @@
 * @param {string} name - The name of the user to greet
 * @returns {null}
 */
-function logGreeting() {
-
-}
+function logGreeting(Avery) {
+  if (Hello)
+  return ("Avery");
+} console.log("Hello Avery!")
 
 /** Returns a custom greeting string in the form "Hello Avery!"
 * @param {string} name - The name of the user to greet
 * @returns {string} - The custom greeting
 */
-function greeting() {
+function greeting(Corey) {
+  let str = "Hey There";
+  return "Hey There " + "Corey";
```

Comment:
This is close. The parameter of a function should be treated like a declared variable. Variables should also be camelCase. If your variable is called corey, if you want to use it you write corey not "corey". "corey" is just a string.